winterfell/webapp/src/js/noangular/signin.js

```diff
@@ -49,15 +74,15 @@ $(document).ready(function(){
     $('.inputs').removeClass('vfi-show');
   });
 
-  $('#signup-view .submit-btn').click(function(e) {
+  $('#signup-view .inputs form').submit(function(e) {
```

Comment:
Nice, my understanding is we want login/signup to be as close to a vanilla html form as possible. Could we maybe just use the `<form onsubmit=` attribute instead of jquery? I think that will help some browsers. This page is pretty simple we might not need jquery at all.